schemas/dictionary.json

```diff
@@ -545,6 +548,14 @@
         "{\n  \"header\": true\n}\n"
       ]
     },
+    "commentChar": {
```

Comment:
@roll any idea why csv ddf spec is being touched here?

Reply:
@rufuspollock 
My guess (I had added it to the description):
> It also includes an addition of commentChar to schemas which probably was forgotten to built previously.

Reply:
It popped up when I ran `npm run build`

Reply:
Ah, thanks 👍